src/Turbo/doc/index.rst

```diff
@@ -453,7 +454,7 @@ Let's create our chat::
                 $form = $emptyForm;
             }
 
-            return $this->render('chat/index.html.twig', [
+            return $this->renderForm('chat/index.html.twig', [
```

Comment:
renderForm() is deprecated since Symfony 6.2, render() is now the way to go.

Reply:
@jmsche thanks